packages/nodes-base/nodes/KoboToolbox/descriptions/SubmissionQueryOptions.ts

```diff
@@ -0,0 +1,79 @@
+import {
+	INodeProperties,
+} from 'n8n-workflow';
+
+export const submissionQueryOptions = [
+	{
+		displayName: 'Start',
+		name: 'start',
+		type: 'number',
+		displayOptions: {
+			show: {
```

Comment:
I must say I'm not really convinced about this split menu structure, keeping `filters`, `sort`, and `fields` under the same Query Options collection make more sense to me, as they're the 3 typical options you'd expect from a query operation.

Besides they all have the same `displayOptions` (when I try to move some under the generic options collection, I'm getting a `Could not resolve parameter depenencies. Max iterations reached!` error...)...

Also I've kept the `start` parameter because I'm certain that some workflows will want to implement their own pagination (somehow... it might be tough... but some forms will have millions of submissions, we can't load them all in memory...).